core/trino-main/src/main/java/io/trino/execution/scheduler/policy/PhasedExecutionSchedule.java

```diff
@@ -79,8 +81,14 @@
      * Graph representing topology between fragments (e.g. child -> parent relationship).
      */
     private final DirectedGraph<PlanFragmentId, FragmentsEdge> fragmentTopology;
+    private final List<PlanFragmentId> sortedFragments = new ArrayList<>();
     private final Map<PlanFragmentId, StageExecution> stagesByFragmentId;
-    private final Set<StageExecution> activeStages = new HashSet<>();
+    private final Set<StageExecution> activeStages = new LinkedHashSet<>();
+
+    /**
```

Comment:
nit: make it non-javadoc comment

Reply:
Then `{@link PhasedExecutionSchedule#init(Collection)}` wouldn't be managed by IntelliJ refactors

Reply:
Meh - not enough of justification IMO. As a javadoc this comment does not make sense. 
But I will not fight over that.